packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/utils.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+/// A widget for positioning sized widgets that follows 2D grid layout:
+///      | top    |
+/// left | center | right
+///      | bottom |
+@immutable
+class GridPositioned extends StatelessWidget {
```

Comment:
maybe call this BorderLayout or similar for consistency with BorderLayout in Java? https://docs.oracle.com/javase/7/docs/api/java/awt/BorderLayout.html
GridLayout makes me think of a 2d grid which is a bit different.

Reply:
this is a very nice suggestion! :-)
I was confused on how to name this.